.release-please-manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   ".": "1.0.0",
   "packages/bff": "1.0.0",
-  "packages/dialogporten-types-generated": "1.0.0",
+  "packages/bff-types-generated": "1.0.0",
```

Comment:
kunne dette vert `packages/*` eller noe? Eller holder det med bare `.`? Eller kan CI'en gennerere denne filen basert på et filter som en del av CI jobben istedenfor å ha den committed i repoet?

Blir så mye rot om det skal være hundre plasser man må oppdatere hver gang man legger til eller fjerner en pakke. `package.json` filene burde være "master dataen".

Følgende kommando lister alle pnpm workspaces i monorepoet:

```
pnpm m ls --depth -1 --json
```

Reply:
Ja sant! Burde absolutt holde. Kan følge opp med en annen PR for det?